sonic_data_client/db_client.go

```diff
@@ -670,6 +670,8 @@ func emitJSON(v *map[string]interface{}) ([]byte, error) {
 func tableData2Msi(tblPath *tablePath, useKey bool, op *string, msi *map[string]interface{}) error {
 	redisDb := Target2RedisDb[tblPath.dbNamespace][tblPath.dbName]
 
+	// Start a Redis transaction to avoid consistency introduced by concurrent cmd occured.
```

Comment:
>transaction [](http://example.com/codeflow?start=18&length=11)

The transaction is overkill since it blocked other redis reading/writing for a long time. Suggest only handle exception on HGet.

Reply:
Changed into ignore instead of blocking the whole serial of operations.